mmcv/ops/nms.py

```diff
@@ -15,13 +14,21 @@
 class NMSop(torch.autograd.Function):
 
     @staticmethod
-    def forward(ctx, bboxes, scores, iou_threshold, offset):
+    def forward(ctx, bboxes, scores, iou_threshold, score_threshold, max_num,
```

Comment:
`score_threshold` and `max_num` arguments should be added after `offset`  to keep the compatibility.